use-cases/demos/Makefile

```diff
@@ -10,36 +10,38 @@
 DOCKER_COMPOSE ?= docker-compose.yml
 PIPELINE_COUNT ?= 1
 DEPENDENCY_COMPOSE ?= add_dependent_services.yml
+RETAIL_USE_CASE_ROOT ?= ../..
+RESULTS_DIR ?= ../../results
 
 build:
 	docker build --tag python-demo:dev -f Dockerfile .
 
 gen_classification:
-	@PIPELINE_COUNT=${PIPELINE_COUNT} docker compose -f $(DEPENDENCY_COMPOSE) -f add_demo_classification.yml convert > $(DOCKER_COMPOSE)
+	@PIPELINE_COUNT=${PIPELINE_COUNT} @RETAIL_USE_CASE_ROOT=$(RETAIL_USE_CASE_ROOT) @RESULTS_DIR=$(RESULTS_DIR) docker compose -f $(DEPENDENCY_COMPOSE) -f add_demo_classification.yml convert > $(DOCKER_COMPOSE)
```

Comment:
remove @ symbol for REATIL_USE_CASE_ROOT and RESULTS_DIR as it is not working when I tested it